Cargo.toml

```diff
@@ -97,6 +103,7 @@ kube = "0.88.1"
 rcgen = "0.12.1"
 reqwest = { version = "0.11.24", default-features = false, features = ["rustls-tls"] }
 rocksdb = "0.21.0"
+rustc-hex = "2.1.0"
```

Comment:
I guess this is needed because the `ethers-*` crates implement its `ToHex`/`FromHex` traits? Would it make sense to get rid of `hex` then, and use `rustc-hex` everywhere? (Not in this PR, of course.)

Reply:
Ok

Reply:
(Actually, it looks like Alloy will use `hex`: https://github.com/alloy-rs/core/blob/main/crates/primitives/Cargo.toml; so maybe we'll rather get rid of `rustc-hex` in the future. :shrug:)